website/content/docs/internals/architecture.mdx

```diff
@@ -6,116 +6,84 @@ description: Learn about the internal architecture of Vault.
 system (KMS) or hardware security module (HSM) to increase security.
 
-The configuration of those audit devices, auth methods, and secrets engines must
-be stored in Vault since they are security sensitive. Only users with the
-correct permissions should be able to modify them, meaning they cannot be
-specified outside of the barrier. By storing them in Vault, any changes to them
-are protected by the ACL system and tracked by audit logs.
+All security-sensitive configurations for audit devices, auth methods, and secrets engines must be stored in Vault. Users with permissions can modify them and cannot be specified outside of the barrier. By storing them in Vault, changes are protected by the ACL system and tracked by audit logs.
 
-After the Vault is unsealed, requests can be processed from the HTTP API to the
-Core. The core is used to manage the flow of requests through the system,
+Requests may be processed from the HTTP API to the core once Vault is unsealed.
+The core manages the flow of requests through the system,
 enforce ACLs, and ensure audit logging is done.
 
-When a client first connects to Vault, it needs to authenticate. Vault provides
-configurable auth methods providing flexibility in the authentication mechanism
-used. Human friendly mechanisms such as username/password or GitHub might be
+When a client first connects to Vault, the client needs to authenticate. Vault provides
+configurable auth methods and offers flexibility within the authentication mechanism
+used. Mechanisms such as username/password or GitHub may be
 used for operators, while applications may use public/private keys or tokens to
-authenticate. An authentication request flows through core and into an auth
-method, which determines if the request is valid and returns a list of
+authenticate. An authentication request that flows through the core and into an auth
+method determines if the request is valid and returns a list of
 associated policies.
 
 Policies are just a named ACL rule. For example, the "root" policy is built-in
-and permits access to all resources. You can create any number of named policies
-with fine-grained control over paths. Vault operates exclusively in a allowed-access
-mode, meaning that unless access is explicitly granted via a policy, the action
-is not allowed. Since a user may have multiple policies associated, an action is
-allowed if any policy permits it. Policies are stored and managed by an internal
-policy store. This internal store is manipulated through the system backend,
+and permits access to all resources. You may create any number of named policies
+with fine-grained control over paths. Vault operates exclusively in an allowed-access mode, meaning the action is not allowed unless access is granted via a policy explicitly. Since a user may have multiple policies associated, actions are allowed when policy permits. Policies are stored and managed by an internal
```

Comment:
Actually, there's more than allow, so let's removed "exclusively" part. 

```suggestion
with fine-grained control over paths. Vault operates in an allowed-access mode, meaning the action is not allowed unless access is granted via a policy explicitly. Since a user may have multiple policies associated, actions are allowed when policy permits. Policies are stored and managed by an internal
```